../libshumate-1.0.0.beta/shumate/shumate-vector-renderer.c:465:3: error: void function 'shumate_vector_renderer_set_sprite_sheet_data' should not return a value [-Wreturn-type]
  return FALSE;

Index: shumate/shumate-vector-renderer.c
--- shumate/shumate-vector-renderer.c.orig
+++ shumate/shumate-vector-renderer.c
@@ -465,7 +465,6 @@ shumate_vector_renderer_set_sprite_sheet_data (Shumate
                SHUMATE_STYLE_ERROR,
                SHUMATE_STYLE_ERROR_SUPPORT_OMITTED,
                "Libshumate was compiled without support for vector tiles.");
-  return FALSE;
 #endif
 }
 
